sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_models.py

```diff
@@ -1164,8 +1164,49 @@ def __repr__(self):
             .format(self.positive, self.neutral, self.negative)[:1024]
 
 
-class EntitiesRecognitionTask(DictMixin):
-    """EntitiesRecognitionTask encapsulates the parameters for starting a long-running Entities Recognition operation.
+class AnalyzeBatchActionsType(str, Enum):
+    """The type of batch action that was applied to the documents
+    """
+    RECOGNIZE_ENTITIES = "recognize_entities"  #: Entities Recognition action.
+    RECOGNIZE_PII_ENTITIES = "recognize_pii_entities"  #: PII Entities Recognition action.
+    EXTRACT_KEY_PHRASES = "extract_key_phrases"  #: Key Phrase Extraction action.
+
+class AnalyzeBatchActionsResult(DictMixin):
```

Comment:
this is the result object we return to users. `begin_analyze_batch_actions` returns a list of `AnalyzeBatchActionsResult`